grpc-spring-boot-starter-demo/src/test/java/org/lognet/springboot/grpc/HalfCloseInterceptor.java

```diff
@@ -0,0 +1,25 @@
+package org.lognet.springboot.grpc;
+
+import io.grpc.ForwardingServerCallListener;
+import io.grpc.Metadata;
+import io.grpc.ServerCall;
+import io.grpc.ServerCallHandler;
+import io.grpc.ServerInterceptor;
+
+@GRpcGlobalInterceptor
```

Comment:
Why the `HalfCloseInterceptor` is registered as standalone interceptor ? I believe it should be abstract to serve as infrastructure support.